test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+describe RecipesController do
+  describe "#home" do
+    it "successfully loads home page" do
+      get root_path
+      must_respond_with :ok
+    end
+
+    it "stays on home page if search term is an empty string" do
+      VCR.use_cassette("recipes") do
+        get root_path(search: " ")
+        must_respond_with :ok
+      end
+    end
+  end
+
+  describe "#index" do
+
+    it "successfully loads recipe results for any search term" do
```

Comment:
It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?